src/components/ClusterManager.js

```diff
@@ -464,12 +473,19 @@ export default ajaxCaller(class ClusterManager extends PureComponent {
   componentDidUpdate(prevProps) {
     const prevCluster = _.last(_.sortBy('createdDate', _.remove({ status: 'Deleting' }, prevProps.clusters))) || {}
     const cluster = this.getCurrentCluster() || {}
+    const twoMonthsAgo = _.tap(d => d.setMonth(d.getMonth() - 2), new Date())
+    const createdDate = new Date(cluster.createdDate)
 
     if (cluster.status === 'Error' && prevCluster.status !== 'Error' && !_.includes(cluster.id, errorNotifiedClusters.get())) {
       notify('error', 'Error Creating Notebook Runtime', {
         message: h(ClusterErrorNotification, { cluster })
       })
       errorNotifiedClusters.update(Utils.append(cluster.id))
+    } else if (createdDate > twoMonthsAgo && !_.includes(cluster.id, outdatedNotifiedClusters.get())) {
```

Comment:
I just got this warning on a new cluster. This should probably be `createdDate < twoMonthsAgo`, right?